config/locales/en/coronavirus_local_restrictions.yml

```diff
@@ -3,12 +3,12 @@ en:
     errors:
       no_postcode:
```

Comment:
I feel like this could benefit from being renamed to `postcode_not_found` to match the condition. The current name makes me think it refers to a blank postcode, which is not what this condition means anymore.